frontend/src/components/FormField/index.module.css

```diff
@@ -3,4 +3,5 @@
   grid-template-rows: repeat(2, auto);
   grid-gap: 12px;
   width: 100%;
+  margin-bottom: 32px;
```

Comment:
Hmm nope. This is a self-contained component. You should be able to drop it anywhere. Meaning, that you probably have to look at the parent. Or create a `FormFieldGroup` / `FormFieldLayout` component.